ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -155,22 +155,27 @@
           {{#each @model.attributes.signing_algorithms as |algorithm i|}}
             <body.row as |row|>
               <row.cell>
-                <form.select
+
+                <Hds::Form::Select::Field
                   @value={{algorithm.value}}
-                  @error={{@model.errors.attributes_signing_algorithms}}
-                  @contextual={{true}}
-                  @onChange={{fn (mut algorithm.value)}}
-                  as |field|
+                  @isInvalid={{@model.errors.attributes_signing_algorithms}}
+                  disabled={{form.disabled}}
+                  {{on 'change' (set-from-event algorithm 'value')}}
+                  as |F|
```

Comment:
personally, I find something like this `as |field|` more readable. Thoughts on this?

Reply:
I agree.  However, [HDS uses this syntax](https://design-system-components-hashicorp.vercel.app/components/form/select).  For consistency with the HDS docs and for consistency across products and time, we should probably align with HDS.